src/app/models/user.server.model.ts

```diff
@@ -71,6 +71,9 @@ const compileUserModel = (db: Mongoose) => {
       betaFlags: {
         sgid: Boolean,
       },
+      flags: {
+        lastSeenFeatureUpdateDate: String,
```

Comment:
should this be a date instead? or even number (for timestamp?) string type may be lossy

Reply:
Initially I thought it would be easier to store it as string since the date stored in the JSON file would be as a string as well. But now that I think about it, if we need to do comparison of dates, we would need to use an external date library like `moment` to handle the comparison. Since the date type in Mongo is represented as UTC datetime we can handle the standardisation of date with `moment` as well. I will update it to date type instead.